module/tools/RoboCupConfiguration/src/RoboCupConfiguration.cpp

```diff
@@ -438,21 +459,22 @@ namespace module::tools {
         mvprintw(2, 2, "Networking");
         attroff(A_ITALIC);
         mvprintw(4, 2, ("Hostname        : " + hostname).c_str());
-        mvprintw(5, 2, ("Wifi Interface  : " + wifi_interface).c_str());
-        mvprintw(6, 2, ("IP Address      : " + ip_address).c_str());
-        mvprintw(7, 2, ("Player ID       : " + std::to_string(player_id)).c_str());
-        mvprintw(8, 2, ("Team ID         : " + std::to_string(team_id)).c_str());
-        mvprintw(9, 2, ("SSID            : " + ssid).c_str());
-        mvprintw(10, 2, ("Password        : " + password).c_str());
+        mvprintw(5, 2, ("Robot Name      : " + robot_name).c_str());
+        mvprintw(6, 2, ("Wifi Interface  : " + wifi_interface).c_str());
+        mvprintw(7, 2, ("IP Address      : " + ip_address).c_str());
+        mvprintw(8, 2, ("SSID            : " + ssid).c_str());
+        mvprintw(9, 2, ("Password        : " + password).c_str());
 
         attron(A_ITALIC);
         mvprintw(2, 40, "Game Configuration");
         attroff(A_ITALIC);
-        mvprintw(5, 40, ("Position: " + std::string(robot_position)).c_str());
+        mvprintw(5, 40, ("Player ID: " + std::to_string(player_id)).c_str());
+        mvprintw(6, 40, ("Team ID  : " + std::to_string(team_id)).c_str());
+        mvprintw(7, 40, ("Position : " + std::string(robot_position)).c_str());
 
         std::stringstream ready_string{};
         ready_string << ready_position.transpose();
-        mvprintw(6, 40, ("Ready   : " + ready_string.str()).c_str());
+        mvprintw(8, 40, ("Ready    : " + ready_string.str()).c_str());
```

Comment:
Here as well, what does 2,5,6,7,8,40 stand for?

Reply:
I did keep row numbers in as they're more to do with being sequential on each line.